client/src/components/Tickets/TicketModal.js

```diff
@@ -34,6 +34,20 @@ const formSchema = Yup.object().shape({
   reply: Yup.string().required('Required')
 });
 
+const Attachments = ({ attachments }) => (
+  <div>
+    Attachments:
+    <ul>
+      {attachments.map(attachmentName => (
+        <li> {attachmentName} </li>
+      ))}
+    </ul>
+  </div>
+);
+Attachments.propTypes = {
+  attachments: PropTypes.string.isRequired
```

Comment:
Looks like `attachments` is an array of objects, not a string. Can you change this to reflect that?